### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by shaunpersad**

### CODE REVIEW

File: content/workers/platform/changelog.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ rss: file
 
 # Changelog
 
+## 2023-03-24
+
+- Fixed a bug that prevented the "Administrator Read-Only" and "Workers Tail Read" roles from successfully tailing workers.
```
Comment (tanushree-sharma) [inline content/workers/platform/changelog.md]:
```suggestion
- Fixed a bug in Wrangler tail and and live logs on the dashboard that prevented the "Administrator Read-Only" and "Workers Tail Read" roles from successfully tailing workers.
```

**Comment by tanushree-sharma** [comment]
Not sure why previews are not showing. I made a minor suggestion on the copy, otherwise looks good.

**Comment by pedrosousa** [comment]
> Not sure why previews are not showing. I made a minor suggestion on the copy, otherwise looks good.

"Commits/PRs from forked repositories will not create a preview." ([from the docs](https://developers.cloudflare.com/pages/platform/known-issues/#enabling-access:~:text=PRs%20from%20forked%20repositories))

File: content/workers/platform/changelog.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ rss: file
 
 # Changelog
 
+## 2023-03-24
+
+- Fixed a bug in Wrangler tail and and live logs on the dashboard that prevented the "Administrator Read-Only" and "Workers Tail Read" roles from successfully tailing workers.
```
Comment (deadlypants1973) [inline content/workers/platform/changelog.md]:
```suggestion
- Fixed a bug in Wrangler tail and and live logs on the dashboard that prevented the Administrator Read-Only and Workers Tail Read roles from successfully tailing Workers.
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by deadlypants1973** [comment]
Merged with approval from @shaunpersad

**Pull request merged**